elao.app.docker/Makefile.dist

```diff
@@ -3,30 +3,31 @@
 -include .manala/Makefile
 
 define setup
-	$(DOCKER_MAKE) install build
+	$(DOCKER_MAKE) install db.install build
 endef
 
 ###########
 # Install #
 ###########
 
 ## Install application
-install:
-	# # Composer
-	# composer install --ansi --verbose
-	# # Doctrine
-	# bin/console doctrine:database:create --ansi --if-not-exists
-	# # Npm
+install: install.composer install.npm install.yarn
+
+install.composer:
+	# symfony composer install --ansi --verbose
```

Comment:
Using the `symfony` CLI for dev targets, which allows to run the target with proper PHP version, even if not on the container, if the PHP version is installed on the host. Mainly targets projects that can run on both host or Manala container, but it cost nothing to always use it, and would even prevent from running such commands on production servers by mistake, where the binary is not installed.

Reply:
imho, we can start to remove yarn related stuff :)

Reply:
Hagrid

Reply:
phare ouëlle